app/javascript/components/App.js

```diff
@@ -1,38 +1,31 @@
-import React, { Component } from 'react'
-import Header from './components/Header'
-import Footer from './components/Footer'
-import Home from './pages/Home'
-import ApartmentIndex from './pages/ApartmentIndex'
-import ApartmentShow from './pages/ApartmentShow'
-import ApartmentNew from './pages/ApartmentNew'
-import ApartmentEdit from './pages/ApartmentEdit'
-import NotFound from './pages/NotFound'
-import {
-  BrowserRouter as Router,
-  Route,
-  Switch
-} from 'react-router-dom'
+import React, { Component } from "react";
+import Header from "./components/Header";
+import Footer from "./components/Footer";
+import Home from "./pages/Home";
+import ApartmentIndex from "./pages/ApartmentIndex";
+import ApartmentShow from "./pages/ApartmentShow";
+import ApartmentNew from "./pages/ApartmentNew";
+import ApartmentEdit from "./pages/ApartmentEdit";
+import NotFound from "./pages/NotFound";
+import { BrowserRouter as Router, Route, Switch } from "react-router-dom";
 
 class App extends Component {
   render() {
     return (
-      
-        <Router>
-          <Header {...this.props} />
-          <Switch>
-            <Route exact path="/" component={Home} />
-            <Route path="/apartmentindex" component={ApartmentIndex} />
-            <Route path="/apartmentshow" component={ApartmentShow} />
-            <Route path="/apartmentnew" component={ApartmentNew} />
-            <Route path="/apartmentedit" component={ApartmentEdit} />
-            <Route component={NotFound}/>
-          </Switch>
-        </Router>
-        
-        
-  
-    )
+      <Router>
+        <Header {...this.props} />
+        <Switch>
+          <Route exact path="/" component={Home} />
+          <Route path="/apartmentindex" component={ApartmentIndex} />
+          <Route path="/apartmentshow" component={ApartmentShow} />
+          <Route path="/apartmentnew" component={ApartmentNew} />
+          <Route path="/apartmentedit" component={ApartmentEdit} />
+          <Route component={NotFound} />
+        </Switch>
+        <Footer {...this.props} />
```

Comment:
Do you need all of the props for Devise login and logout being brought into your footer?

Reply:
I wanted to test something out after I had an error and forgot to remove it. Thank you for catching it.